### ISSUE TITLE
parentMaterialSampleID

### ISSUE THREAD

**Issue by thomasstjerne** [gbif/rs.gbif.org#37]
We have the field `materialSampleID`  from material sample core on DWC occurrence, even though material sample core is not in production. 

Data publishers from Australia, Norway and Sweden has requested  `parentMaterialSampleID` to be added to dwc:Occurrence.

Proposing this for both DWC occurrence and material sample core

**Comment by thomasstjerne** [comment]
@tucotuco Do you have an opinion on this?

**Comment by dagendresen** [comment]
Maybe the first required step is a **MaterialSample** core?!
With the current **Occurrence** core there is also the **RelatedResource** extension available...

**Comment by timrobertson100** [comment]
This repository is the GBIF repository of schemas. For occurrence and event core definitions the policy is to follow DwC strictly and only deviate when necessary. `recordedByID` and `identifiedByID` are examples where GBIF needed to deviate as DwC decided not to adopt them. Now with the widespread use of those terms I suspect it likely they will be adopted by DwC.

I suggest the best way to proceed would be to propose `parentMaterialSampleID` in DwC and see the outcome of that discussion before considering a change here. See https://github.com/tdwg/dwc/blob/master/.github/CONTRIBUTING.md for details on how to propose the term there.

**Comment by deepreef** [comment]
I agree with @timrobertson100  that the term `parentMaterialSampleID` should be formally proposed in DwC.  I'm willing to do this, but I'll defer to @thomasstjerne if he would like to propose it.  We have implemented this recursive hierarchical relationship within `MaterialSample` for a long time, and it is very useful.

**Comment by timrobertson100** [comment]
If you have the time to, please can you @deepreef? 

DwC is preparing a batch of revisions for public commentary to go out at the end of April, so the sooner the better to make sure it makes the cut.

**Comment by deepreef** [comment]
> If you have the time to, please can you @deepreef?

OK, if no one else proposes it by tomorrow, I will do so.

I may also propose `materialSampleType` at the same time (see [here](https://github.com/tdwg/dwc/issues/314#issuecomment-827891963)).

**Comment by tucotuco** [comment]
> @tucotuco Do you have an opinion on this?

Sorry that I missed this request for feedback. When it comes to Darwin Core proposals, I try to be a good magistrate and listen to the evidence and provide guidance. I fail often. In any case, my DwC convenor self says to propose it using the the new term template as completely, concisely, and clearly laid out as possible, especially if it is to be fast-tracked. The first challenge will be to demonstrate the evidence of demand. It looks like you have evidence from data publishers. It would make a better case to add the evidence that independent parties need to share these data (have consumers).

**Comment by deepreef** [comment]
@tucotuco :  
> propose it using the the new term template as completely, concisely, and clearly laid out as possible, especially if it is to be fast-tracked.

Prompted by @timrobertson100 , I have started drafting this exactly as you describe, including the "concisely...as possible" part [waits for skeptical laughter to subside].  I mention at least two independent parties that have a need to share this information.  Unless someone speaks up wanting to to do this, I will create an issue in the DwC space for this (as well as the proposed new term `materialSampleType`, and some proposals to move other terms from the `Occurrence` class to the `MaterialSample` class.)
